hips/hip-9999.md

```diff
@@ -0,0 +1,86 @@
+---
+hip: 9999
```

Comment:
Can you update this to 0021, update the file name, and update the table of contents in the readme.